docs-ref-toc/top_level_toc.yml

```diff
@@ -1198,6 +1203,11 @@
       landingPageType: Service
       children:
       - 'Microsoft.Azure.Management.TrafficManager*'
+  - name: Translation
```

Comment:
@maririos any reason you put this in 2 locations in the ToC? Looks like you put it under cognitive services and at the root.

![image](https://user-images.githubusercontent.com/9010698/116438477-5509b580-a803-11eb-8690-4d9ab265c8fa.png)

looks similar for text analytics. We should probably only have in one or the other places as I believe some of the docs links look off because of this. https://docs.microsoft.com/en-us/dotnet/api/overview/azure/ai.translation.document-readme-pre?view=azure-dotnet-preview

Reply:
I was following prior art. To your point, I am unclear if they should go in the root or in the Cognitive Services root. I see this happens for the Cognitive Services that have SDKs our team has directly worked on.
Do you know if there is a guideline/rule for this?

Reply:
I don't think there is a guideline. I'd suggest reaching out to the PM for your services and see what they recommend.